Libplanet.Tests/Libplanet.Tests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is there any change needed?

Reply:
Rider repeatedly tries to insert BOM to *.csproj* files.  I'm going to trim this.